app/assets/stylesheets/base.scss

```diff
@@ -1,4 +1,3 @@
-@import "bootstrap-tagsinput";
```

Comment:
Is removal of this a related change?

Reply:
Hm. This isn't related. It may have been accidentally removed while resolving conflicts when I rebased on master. Let me add this back in.

Reply:
@mejackreed Okay. This has been fixed.